pyproject.toml

```
@@ -36,6 +36,10 @@ langchain-openai = { version = "^0.0.5", optional = true}
 google-api-python-client = { version = "2.95.0", optional = true}
 subgrounds = "^1.8.1"
 loguru = "^0.7.2"
+safe-eth-py = "^6.0.0b14"
+eth-account = "0.8.0"
```

```suggestion
eth-account = "^0.8.0"
```

Could be problematic in the future

---

moved back to ^